openmdao/utils/options_dictionary.py

```diff
@@ -44,6 +46,25 @@ def __init__(self, parent_name=None, read_only=False):
         self._parent_name = parent_name
         self._read_only = read_only
 
+        self._all_recordable = True
+
+    def __getstate__(self):
+        """
+        Return state as a dict.
+
+        Returns
+        -------
+        dict
+            State to get.
+        """
+        if self._all_recordable:
+            state = self.__dict__
+            return state
```

Comment:
No need for a temporary variable here, just do `return self.__dict__`

Reply:
Fixed